docs/extensions/index.md

```diff
@@ -99,11 +99,16 @@ If the pre processing operator is a HuggingFace tokenizer, you can also easily g
 
 ```python
 import onnxruntime as _ort
-from transformers import AutoTokenizer
+from transformers import AutoTokenizer, GPT2Tokenizer
 from onnxruntime_extensions import OrtPyFunction, gen_processing_models
 
-tokenizer = AutoTokenizer.from_pretrained('distilbert-base-uncased')
-model = OrtPyFunction(gen_processing_models(tokenizer, pre_kwargs={})[0])
+# SentencePieceTokenizer
```

Comment:
Is it clear which model a user would choose for each tokenizer?

Reply:
there isn't one specific model - depending on the tokenizer there could be multiple options. i only include one for each here as an example.